src/views/Download.vue

```diff
@@ -38,7 +38,7 @@
             verify
             hashes and signatures </a> which you can follow by using any of these
             release-signing <a class="link-color"
-                               href="https://downloads.apache.org/incubator/iotdb/KEYS">
+                               href="https://www.apache.org/dist/incubator/iotdb/KEYS">
```

Comment:
Use `downloads.apache.org` instead of `www.apache.org/dist`
And also all download links in this file.